<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureSet_Dialog</class>
 <widget class="QDialog" name="PictureSet_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>白平衡</string>
   </property>
   <widget class="QLabel" name="R_channelDisp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>   </string>
    </property>
   </widget>
   <widget class="QLabel" name="G_channelDisp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="B_channelDisp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSlider" name="R_channelSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="G_channerlSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>R通道:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>G通道:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>B通道:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="B_channelSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="SetBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
